core/src/crds_value.rs

```diff
@@ -134,13 +176,35 @@ impl EpochSlots {
     }
 }
 
+impl Sanitize for EpochSlots {
+    fn sanitize(&self) -> Result<(), SanitizeError> {
+        if self.wallclock >= MAX_WALLCLOCK {
```

Comment:
do we also want to verify the `Uncompressed/Compressed` inside `epoch_slots.slots` (verify lengths and `num` to avoid issues like: https://github.com/solana-labs/solana/pull/9595) inside the EpochSlots, or is that best left to the processing logic

Reply:
came up in the original pr, opinions are mixed: https://github.com/solana-labs/solana/pull/9706#discussion_r415157399

Reply:
need another PR